libs/libarchfpga/src/read_xml_arch_file.cpp

```diff
@@ -3799,6 +3825,12 @@ static void ProcessSegments(pugi::xml_node Parent,
         /* Get next Node */
         Node = Node.next_sibling(Node.name());
     }
+    /*if no segment with default axis option isn't provided, atleast 1 segment along each x & y axes is needed */
```

Comment:
Can improve this comment.  Should say we need at least one type of segment that applies to each of x- and y-directed wiring.